tests/regression/test_simulate_single_operation.py

```diff
@@ -0,0 +1,65 @@
+import configparser
+import os
+from unittest.mock import patch, MagicMock
+from pay.batch_payer import BatchPayer, FEE_INI, MUTEZ_PER_GAS_UNIT
+from model.reward_log import RewardLog
+from cli.client_manager import ClientManager
+from http import HTTPStatus
+from Constants import PaymentStatus
+
+run_ops_parsed = {
+    "contents": [
+        {
+            "metadata": {
+                "operation_result": {
+                    "status": "applied",
+                    "consumed_gas": "100",
+                    "paid_storage_size_diff": "24",
+                },
+                "internal_operation_results": [{"result": {"consumed_gas": "50"}}],
+            }
+        }
+    ]
+}
+
+
+@patch(
+    "cli.client_manager.ClientManager.request_url_post",
+    MagicMock(return_value=(HTTPStatus.OK, run_ops_parsed)),
+)
+def test_simulate_single_operation():
```

Comment:
@jdsika @utdrmac @amzid I added a regression test for the consumed_gas estimation here please update or add it according to the new PR (https://github.com/tezos-reward-distributor-organization/tezos-reward-distributor/pull/411).